sql/src/main/java/io/crate/analyze/AlterTableAnalyzer.java

```diff
@@ -69,8 +70,17 @@ AlterTableRenameAnalyzedStatement analyzeRename(AlterTableRename node, SessionCo
             throw new IllegalArgumentException("Target table name must not include a schema");
         }
 
-        RelationName relationName = RelationName.of(node.table(), sessionContext.defaultSchema());
-        DocTableInfo tableInfo = schemas.getTableInfo(relationName);
+        RelationName relationName;
+        DocTableInfo tableInfo;
+        if (node.blob()) {
+            relationName = new RelationName(BlobSchemaInfo.NAME, node.table().getName().getParts().get(0));
```

Comment:
What if someone used `alter blob table blob.xy` ? Wouldn't that result in the wrong table name here?
Could also change it to just do the `relationName` assignment with the branches. The `tableInfo` assignment is the same in both cases.